### PR TITLE
company detail

### PR DESCRIPTION
**Pull request by DongHien0896**
code producers detail:
https://edu-redmine.framgia.vn/issues/1304

### CODE REVIEW

**Comment by DongHien0896** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -23,24 +22,25 @@
...
-        <activity android:name="com.framgia.hien.moviedb.screen.person.PersonActivity"
+        <activity
+            android:name="com.framgia.hien.moviedb.screen.person.PersonActivity"
             android:screenOrientation="portrait"
             android:theme="@style/CustomAppTheme" />
+        <activity android:name="com.framgia.hien.moviedb.screen.company.CompanyActivity"></activity>
     </application>
-</manifest>
+
+</manifest>
```
Comment (tuannguyen38) [inline app/src/main/AndroidManifest.xml]: need an endline

File: app/src/main/java/com/framgia/hien/moviedb/data/repository/CompanyRepository.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import com.framgia.hien.moviedb.data.source.remote.CompanyRemoteDataSource;
+
+import io.reactivex.Maybe;
+
+public class CompanyRepository implements CompanyRemoteDataSource.RemoteDataSource {
+
+    private static CompanyRepository sInstance;
+    private CompanyRemoteDataSource mRemoteDataSource;
+
+    public CompanyRepository(CompanyRemoteDataSource dataSource) {
```
Comment (tuannguyen38) [inline app/src/main/java/com/framgia/hien/moviedb/data/repository/CompanyRepository.java]: private constructor

**Comment by DongHien0896** [comment]
ready

**Comment by DongHien0896** [comment]
ready

**Comment by DongHien0896** [comment]
ready

**Pull request merged**